src/GuiBase/RadiumWindow/SimpleWindow.cpp

```diff
@@ -0,0 +1,85 @@
+#include <GuiBase/RadiumWindow/SimpleWindow.hpp>
+
+#include <Engine/Renderer/Renderers/ForwardRenderer.hpp>
+#include <GuiBase/SelectionManager/SelectionManager.hpp>
+#include <GuiBase/TreeModel/EntityTreeModel.hpp>
+#include <GuiBase/Viewer/CameraManipulator.hpp>
+#include <GuiBase/Viewer/Viewer.hpp>
+
+namespace Ra {
+using namespace Gui;
+using namespace Engine;
+
+namespace GuiBase {
+
+SimpleWindow::SimpleWindow( QWidget* parent ) : MainWindowInterface( parent ) {
+    if ( objectName().isEmpty() ) setObjectName( QString::fromUtf8( "RadiumSimpleWindow" ) );
+    resize( 800, 640 );
```

Comment:
I would rather have the size given as parameter of the constructor, with these default values (e.g., if I want to use it on Qt for IOS).